### PR TITLE
[WIP] [Glimmer2] Unskip some test-helper tests

### PR DESCRIPTION
**Pull request by nickiaconis**
The fix for unskipping `'fillIn' focuses on the element` is currently breaking the HTMLBars version of the test, but I haven't been able to figure out why yet.

### CODE REVIEW

File: packages/ember-testing/tests/helpers_test.js [hunk 1]
```diff
@@ -656,21 +661,21 @@ test('`fillIn` takes context into consideration', function() {
...
         ok(true, 'focusIn event was triggered');
       }
     }
-  });
+  }));
 
-  App.IndexView = EmberView.extend({
-    template: compile('<div id="parent">{{input type="text" id="first" focus-in="wasFocused"}}</div>')
-  });
+  App.register('template:components/fill-in', compile('<div id="parent"><input type="text" id="first" onfocusin={{action "wasFocused"}}></div>'));
```
Comment (chadhietala) [inline packages/ember-testing/tests/helpers_test.js]: Can you change this to use `{{input}}` and set this test to `test` from from `skip-it-glimmer`?

**Comment by locks** [comment]
This requires a rebase at the moment.

**Comment by Serabe** [comment]
@chadhietala I see this PR referenced by you (#13625) in a closed PR that seems superseded by #13642. Can this be closed?

Thank you!

**Comment by rwjblue** [comment]
Ya, I think so.  All of the tests have been ported.

**Pull request closed without merging**